docker/initial-data.json

```diff
@@ -112,7 +112,7 @@
         "conference_stream_poster_url": "",
         "conference_open_microphone": true,
         "conference_open_video": true,
-        "conference_auto_connect_next_speakers": true,
+        "conference_auto_connect_next_speakers": 1,
```

Comment:
this should initially be 0. Per default, you do not want that speakers automatically connect into jitsi

Reply:
Ok sure, I just converted the `true` to a 1